tests/screenshot/apps/components/ContextualPopupDecorator.js

```diff
@@ -35,14 +42,38 @@ const ContextualPopupDecoratorTests = [
 		<ContextualPopupButton direction="right top" />,
 		<ContextualPopupButton direction="right bottom" />,
 		// *************************************************************
+		// ContextualPopupButton with no arrow
+		// *************************************************************
+		<ContextualPopupButtonNoArrow offset="none" />,
```

Comment:
'offset' prop test is duplicated by line 74~76. I think it need to remove 'offset' prop in below 'NoArrow' test.
```suggestion
		<ContextualPopupButtonNoArrow />,
		
```

Reply:
The tests from 74-76 are for the RTL locale and I think it's useful to have them. Also the 'offset' prop is required for the test.

Reply:
You are right. I missed it.